h/api/models/__init__.py

```diff
@@ -13,6 +13,7 @@
 
 from h.api.models import elastic
 from h.api.models.annotation import Annotation
+from h.api.models.document import ConcurrentDocumentMetaCreateError
```

Comment:
Right, as long as `create_or_update_document_meta()` is public then this does need to be public. If `create_or_update_document_meta()` becomes private then yes